set.mm

```diff
@@ -181253,7 +181277,7 @@ reduced fraction representation (no common factors, denominator
     df-odz $a |- odZ = ( n e. NN |-> ( x e. { x e. ZZ | ( x gcd n ) = 1 } |->
                    inf ( { m e. NN | n || ( ( x ^ m ) - 1 ) } , RR , < ) ) ) $.
 
-    $( Define the Euler phi function (also called _ Euler totient function_),
+    $( Define the Euler phi function (also called _Euler totient function_),
```

Comment:
Here and in iset.mm, I would write `"Euler totient function"`, that is, double quotes instead of formatting, which should be used sparingly.

Reply:
Are ther rules/conventions/practices when to use formatting (bold letters in this case) and when double quotes? Who decided to use formatting sparingly? In my opinion, new terms can be shown in bold the first time they are mentioned.

Reply:
I don't think there are written conventions about that, but I think the closest we stay to plain text, the less the maintenance effort, which is especially important since we do not have much workforce available (for instance: what happens with bold or italic if the rendering engine has to be changed for whatever reason).